### PR TITLE
Holmes release details

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

**Review by aantn — COMMENTED** [review]
@pavangudiwada Can you make more clear the connection between kubernetes-chatgpt-bot and holmes? (Holmes replaces it. We intend to deprecate this project because Holmes is more powerful in every way!)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Introduction
+ ## 🚨This project will be depreciated and replaced by [HolmesGPT](https://github.com/robusta-dev/holmesgpt), our advanced Open Source DevOps assistant. With Holmes you can investigate incidents, triage issues, enrich alerts and much more. Check it out [https://github.com/robusta-dev/holmesgpt](https://github.com/robusta-dev/holmesgpt)
```
Comment (aantn) [inline README.md]: Typo in deprecated

**Review by aantn — APPROVED** [review]
One typo. Other than that lgtm

File: README.md (hunk 1 above)
Comment (pavangudiwada) [reply README.md]: Oops, thanks!

**Pull request merged**